File: 0001 Code Chef/0025 Burger/implementation/0004 java/Burgers.java
import java.util.Scanner;

class Burgers{
    public static void main(String[] args){
        Scanner read = new Scanner(System.in);
        String stringInput;
        stringInput = read.nextLine();
        int test = Integer.parseInt(stringInput);
        while(test > 0){
            stringInput = read.nextLine();
            String[] stringArray = stringInput.split(" ");
            int a = Integer.parseInt(stringArray[0]);
            int b = Integer.parseInt(stringArray[1]);
            if(a > b){
                System.out.println(b);
            }else{
                System.out.println(a);
            }
            test = test - 1;
        }
    }
}

File: 0001 Code Chef/0025 Burger/implementation/0003 c#/Burgers.cs
using System;

class Burgers{
    static void Main(string[] args){
        string stringInput;
        stringInput = Console.ReadLine();
        int test = Convert.ToInt32(stringInput);
        while(test > 0){
            stringInput = Console.ReadLine();
            string[] stringArray = stringInput.Split(" ");
            int a = Convert.ToInt32(stringArray[0]);
            int b = Convert.ToInt32(stringArray[1]);
            if(a > b){
                Console.WriteLine(b);
            }else{
                Console.WriteLine(a);
            }
            test = test - 1;
        }
    }
}